.github/workflows/docker-build.yml

```diff
@@ -84,8 +84,6 @@ jobs:
           dockerfile: Scanner.Dockerfile
           jibImage: scanner-worker
           task: :workers:scanner:jibDockerBuild
-        - jibImage: cli
```

Comment:
Commit message: Maybe argue keeping the JVM target to be able to run the CLI on other targets than the native bin targets?
I guess the JVM has more target platforms as our native binary.

Reply:
That would make sense to me only if we could name such a target that we do not target natively yet. Is there one you have in mind?

Reply:
Linux ARM 64